### ISSUE TITLE
site-thebestshadow

### ISSUE THREAD

**Issue by TheBestShadowI** [TheBestShadowI/TheBestShadow-#1]
[thebestshadow.zip](https://github.com/TheBestShadowI/TheBestShadow-/files/1487554/thebestshadow.zip)
<html>

<audio id="music" autoplay loop > 
<source src="gafgaf.mp3" type="audio/ogg">
</audio>
<style>
    @font-face {
    font-family: DS Pixel Cyr; /* Гарнитура шрифта */
    src: url(fonts/3560.ttf); /* Путь к файлу со шрифтом */
   }
   h1 {
    font-family: DS Pixel Cyr, 'Pixel', cursive;
   }
 </style>
 
 <style>
.dva { 
  background: linear-gradient(green, red);
}
.dva h1 { 
  color: #000;
  background: #fff;
  mix-blend-mode: lighten;
}
</style>

<head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <script src="https://use.fontawesome.com/e2c39a3f30.js"></script>
	<link rel="stylesheet" type="text/css" href="file/css/home.css">
    <link rel="stylesheet" href="file/css/style.css">
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
    <link href="https://file.myfontastic.com/eWT6DHqAn3BdQ3oTWZ397H/icons.css" rel="stylesheet">
    <link rel="icon" href="file/img/favicon.ico">
    
    <title>TheBestShadow</title>
</head>

<body>
<body oncontextmenu="return false">
	<
    	<video autoplay muted loop poster="file/img/background.png" id="bgvid">
       	 	<source src="file/img/background.ogv" type="video/webm">
   	</video>
	>

    <div class="container main" style="margin-top: 15%; z-index: 2; text-align: center;">

        <span id="text" class="text" style="text-align: center; font-family: DS Pixel Cyr; vertical-align: middle; line-height=90px; font-weight: bold; font-size: 50px; z-index: 5;">Hi, I'm TheBestShadow</span>
        <span class="text" style="text-align: center; vertical-align: middle; line-height=90px; font-size: 32px; z-index: 5;" id="mmk">
        <span class="text" id="mkk"style="text-align: center; font-family: DS Pixel Cyr; color: #3bed00; vertical-align: middle; line-height=90px; font-size: 32px; z-index: 5;">Добро пожаловать на сайт </span><br>
		
		
        <hr style="width: 50%; margin-top: 50px;" >

        <ul id="menu" style="font-size: 32px; z-index: 5;color: #3bed00; margin-top: 35px; list-style: none; margin-right:40px;">
            <li><a href="https://github.com/TheBestShadowI"><i class="fa fa-github fa-inverse" Style="color: #3bed00" aria-hidden="true"></i></a></li>
            <li><a href="http://steamcommunity.com/id/THEBESTSHADOWGOOD"><i class="fa fa-steam fa-inverse" Style="color: #3bed00" aria-hidden="true"></i></a></li>
            <li><a href="https://vk.com/thebestshadowtwitch"><i class="fa fa-vk fa-inverse" Style="color: #3bed00" aria-hidden="true"></i></a></li>
            <li><a href="https://www.youtube.com/channel/UClcv8Dqeeybm3HGK3_wxAEw"><i class="fa fa-youtube-play fa-inverse" Style="color: #3bed00" aria-hidden="true"></i></a></li>
        </ul>
    </div>

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
	
<script type="text/javascript" src="file/js/lib/jquery-3.0.0.min.js"></script>
<script type="text/javascript" src="file/js/lib/typed.js"></script>
    
	<script type="text/javascript">
        
            $("#mmk > #mkk").typed({
                strings: ["❤Привет пуся❤","❤Тыкни на иконки чтоб зайти на мои страницы❤","❤Пока пуся❤"],
                typeSpeed: 50,
                backSpeed: 50,
                backDelay: 1000,
                loop: true,
            });
            
            var arrowvisible = true;
            $(window).scroll(function() {
                if ($(window).scrollTop() > 0) {
                    if (arrowvisible) {
                        arrowvisible = false;
                        $("#arrow").addClass("invisible");
                    }
                } else if (!arrowvisible) {
                    arrowvisible = true;
                    $("#arrow").removeClass("invisible");
                }
            });
            
        </script>
<script type="text/javascript">
var base_color="#D50000"
var color1="3bed00"
var color2="ff2800"
var flashspeed=99
var flashingletters=2
var flashingletters2=2
var flashpause=0
var n=0
var color_text=""
message=document.getElementById("text").innerHTML;
if (document.all||document.getElementById){
    color_text='<div align="center"><font color="'+base_color+'" size="6">';
    for (m=0; m<message.length; m++) color_text+='<span id="neonlight'+m+'">'+message.charAt(m)+'</span>'
    color_text+='</font></div>'
    document.getElementById("text").innerHTML = color_text
}
else document.getElementById("text").innerHTML = message

function crossref(number){
    var crossobj=document.all? eval("document.all.neonlight"+number) :
    document.getElementById("neonlight"+number)
    return crossobj
}

function neon(){
    if (n==0){
     for (m=0;m<message.length;m++)
     crossref(m).style.color=base_color
     }
    crossref(n).style.color=color1
    if (n>flashingletters-1) { crossref(n-flashingletters).style.color=color2 }
    if (n>(flashingletters+flashingletters2)-1) {
    crossref(n-flashingletters-flashingletters2).style.color=base_color }
    if (n<message.length-1) { n++ }
    else {
     n=0
     clearInterval(flashing)
     setTimeout("start_color()",flashpause)
     return
     }
}
function start_color(){
    if (document.all||document.getElementById)
    flashing=setInterval("neon()",flashspeed)
}
start_color()
</script>
</body>

</html>
